sandbox/grommet-app/src/pages/layouts/kinds/OpsRamp/ServersTable.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Anchor,
+  Box,
+  Button,
+  DataFilters,
+  DataSummary,
+  Toolbar,
+  DataSearch,
+  Data,
+  DataTable,
+  Pagination,
+} from 'grommet';
+import {
+  StatusCriticalSmall,
+  Upload,
+  SettingsOption,
+  StatusGoodSmall,
+  StatusPlaceholderSmall,
+  StatusUnknownSmall,
+} from 'grommet-icons';
+import opsRamp from '../../../../mockData/opsRamp.json';
+
+import { QuickFilters } from './QuickFilters';
+
+const defaultView = {
+  search: '',
+  sort: { property: 'name', direction: 'asc' as 'asc' | 'desc' },
+  step: 10,
+  properties: {},
+};
+
+interface ServersTableProps {
+  showResultDetails: boolean;
+  setShowResultDetails: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
```

Comment:
Then reuse for `[result, setResult]` & again for `const columns = [`

```suggestion
type Node = {
  name: string;
  'ip address': string;
  make: string;
  model: string;
  state: string;
}
```